### DISCUSSION TITLE
Outdoor panel support?

### DISCUSSION THREAD

**Discussion by srcastor** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#244] [Q&A] [answered]
I read somewhere outdoor panels must be faster scan due to the higher brightness, in fact, I am not able to find 64x64 1/32 panels, they are all 1/16.
So I am asking if anyone knows any outdoor model that works with esp32 or has managed to run a 32x32 1/16 panel, specifically I am looking for a P3 64x64

**Comment by board707** [comment, marked as answer]
@srcastor, it seems some misunderstanding in your question. The reason for the high brightness of the outdoor panels is precisely in their lower scan, so it obviously that you cant find outdoor 64x64 1/32, all outdoors that size only 1/16 1/8 and so on

**Comment by mrcodetastic** [comment]
Refer to: https://github.com/mrfaptastic/ESP32-HUB75-MatrixPanel-I2S-DMA/issues/204
